spring-cloud-gcp-docs/src/main/asciidoc/datastore.adoc

```diff
@@ -386,15 +386,50 @@ This feature is not supported yet.
 
 ==== Custom GQL query methods
 
-This feature is not supported yet.
+Custom GQL queries can be mapped to repository methods in one of two ways:
+
+ * `namedQueries` properties file
+ * using the `@Query` annotation
+
+===== Query methods with annotation
+
+Using the `@Query` annotation:
+
+The names of the tags of the GQL correspond to the `@Param` annotated names of the method parameters.
```

Comment:
Documentation should list which types are supported for the parameters.

Reply:
@meltsufin change made, PTAL.